model_analyzer/config/input/config_command_profile.py

```diff
@@ -1055,9 +1055,12 @@ def _preprocess_and_verify_arguments(self):
             logger.warning(
                 f"--export-path not specified. Using {self._fields['export_path'].default_value()}"
             )
-        elif self.export_path and not os.path.isdir(self.export_path):
+        elif os.path.exists(self.export_path) and os.path.isfile(
```

Comment:
Why would this change from "not isdir" to "isfile" when the error message is saying "this is not a dir".

Reply:
Fixed